tests/shocktube_cma.in

```diff
@@ -0,0 +1,25 @@
+# *****************************************************************
+# Problem size and geometry
+# *****************************************************************
+geometry.prob_lo     =  0.0  0.0  0.0 
+geometry.prob_hi     =  1.0  1.0  1.0
+geometry.is_periodic =  0    1    1
+
+# *****************************************************************
+# VERBOSITY
+# *****************************************************************
+amr.v              = 1       # verbosity in Amr
+
+# *****************************************************************
+# Resolution and refinement
+# *****************************************************************
+amr.n_cell          = 1024 16 16
+amr.max_level       = 1     # number of levels = max_level + 1
+amr.blocking_factor = 16    # grid size must be divisible by this
+
+do_reflux = 1
+do_subcycle = 1
+
+plotfile_interval = -1
+cfl = 0.6
+hydro.reconstruction_order = 3
```

Comment:
Can you turn on artificial viscosity here? This is to make sure that any changes to artificial viscosity in the future don't break CMA or vice versa.